### DISCUSSION TITLE
Buzzer Sounds and Visual

### DISCUSSION THREAD

**Discussion by JasmolSD** [joshzcold/Cold-Family-Feud#92] [Ideas]
Hi Josh, huge fan of the app! It's great! One question:
-  Are there plans to add a buzzer sound effect and indicator on the game window so people know which team buzzed in first? 
 
I know there's a ranking on the devices and the host screen but how about on the game window so it lights up the team name for whichever team buzzed in first?

**Comment by joshzcold** [comment]
That's a great suggestion 👍

I have a 2.0 release coming soon and I'll see if I can get something like this in within that release.

Shouldn't be too hard since I already get that buzzer data on the game UI.

**Comment by ItzEarthy** [comment]
I second this! I also suggest adding the fast money music when you hit the start timer.

**Comment by joshzcold** [comment]
A buzzer sound and popup now happens on the game screen when users buzz in https://github.com/joshzcold/Friendly-Feud/pull/100
